device/mellanox/x86_64-nvidia_sn2201-r0/thermal_policy.json

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/thermal_policy.json
```

Comment:
please check path. might be that you need extra ../ as in other links

Reply:
I verified, the path is valid.